internal/pkg/client/cache/net_test.go

```diff
@@ -34,3 +34,30 @@ func TestNet(t *testing.T) {
 		})
 	}
 }
+
+func TestNetImageExists(t *testing.T) {
+	tests := []struct {
+		name     string
+		sum      string
+		path     string
+		expected bool
+	}{
+		{"empty data", "", "", true},
+		{"invalid data", "not a SHA sum", "not an image", false},
```

Comment:
it probably makes sense to test for an invalid hash and and invalid path in two separate test cases

Reply:
I personally do not see the true benefit since "" is a valid hash and "" a valid path (maybe the semantics could be clarified outside of this issue?). That said, I totally see your point and will add the tests for completeness.